### PR TITLE
Added myself

### PR DESCRIPTION
**Pull request by mschenker**

### CODE REVIEW

File: assets/data/experts.json [hunk 1]
```diff
@@ -2020,5 +2020,20 @@
...
+		"picture": "https://media.licdn.com/mpr/mpr/shrinknp_400_400/AAEAAQAAAAAAAAvdAAAAJDg3Y2IxMDI1LWIyY2ItNDljYS1hMzg3LTIwN2Q1MzRlNzAzYg.jpg",
+		"linkedin": "https://www.linkedin.com/in/matthias-schenker-274623a/",
+		"twitter": "https://twitter.com/MattSchenker",
+		"github": "https://github.com/mschenker",
+		"geonet": "https://community.esri.com/people/m.schenkeresri-ch-esridist",
+		"arcgisonline": null,
+		"stackoverflow": null,
+		"stackexchange": null,
+		"youtube": null,
+		"blog": "",
```
Comment (hhkaos) [inline assets/data/experts.json]: Would you mind you change this empty string for a `null` value? thx!!

**Comment by hhkaos** [comment]
Great @mschenker !! thx!, could you make [this change](https://github.com/esri-es/arcgis-experts/pull/13/commits/269ba96db2a7b2e480ea78fa77b2d47063bb1f75#r160442409)?

**Review by mschenker — COMMENTED** [review]
Looks OK to me now

**Comment by hhkaos** [comment]
Great!, that's it :D, thx!

**Pull request merged**